lib/index.js

```diff
@@ -192,6 +193,14 @@ api.set = async ({
   }
   if(secret) {
     rValue.secret = secret;
+    // produces a URL like `otpauth://totp/...`
+    rValue.otpAuthUrl = authenticator.keyuri(
+      // user identifier
+      email || account,
+      // a service name, `undefined` is OK if `clientId` was not passed in
+      clientId,
```

Comment:
If we put `clientId` here, the totp token will only work with that specific client ... I'm not convinced we want that. Is that intentional? It is, of course, optional (if no `clientId` is passed) ... but is that a feature we want to support?

Reply:
Sounds like this should be `appId` or `serviceId` instead (a new param).

Reply:
Done.